src/unstract/adapters/embedding/ollama/src/static/json_schema.json

```diff
@@ -18,6 +18,12 @@
       "default": "mxbai-embed-large",
       "description": "Provide the name of the model to use for embedding. Example: mxbai-embed-large"
     },
+    "base_url": {
+      "type": "string",
+      "title": "Base URL",
+      "default": "",
+      "description": "Provide the base URL where Ollama server is running."
```

Comment:
@gaya3-zipstack Should we give a default value for this param?

Reply:
Yes. I think that is good. I will make it http://localhost:11434. Thank you for calling this out @harini-venkataraman

Reply:
@gaya3-zipstack but the localhost will make sense only when they are running the backend locally for development.

I think it would be the best to leave it empty. Or `docker.host.internal` so that it would be easier for OSS users. I will suggest same in case of OLLAMA LLM adapter also.

Reply:
Ok. Let me do that @ritwik-g

Reply:
@gaya3-zipstack also add this as required.

Reply:
Added the sample URL to both LLM and embedding